src/content/docs/it/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,423 @@
+---
+title: Aggiorna alla v2 di Astro
+description: Come aggiornare il tuo progetto all'ultima versione di Astro.
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+
+
+Questa guida ti aiuterà a passare da Astro v1 ad Astro v2.
+
+Hai bisogno di aggiornare un vecchio progetto a v1? Vedi la nostra [vecchia guida alla migrazione](/it/guides/upgrade-to/v1/).
+
+## Aggiorna Astro
+
+Aggiorna la versione di Astro del tuo progetto all'ultima versione usando il tuo package manager. Se stai usando le integrazioni di Astro, aggiorna per favore anche queste ultime alla versione più recente.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # Upgrade to Astro v2.x
+  npm install astro@latest
+  
+  # Example: upgrade React and Tailwind integrations
+  npm install @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # Upgrade to Astro v2.x
+  pnpm install astro@latest
+
+  # Example: upgrade React and Tailwind integrations
+  pnpm install @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # Upgrade to Astro v2.x
+  yarn add astro@latest
+  
+  # Example: upgrade React and Tailwind integrations
+  yarn add @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+## Astro v2.0 Cambiamenti principali
+
+Astro v2.0 include alcuni cambiamenti importanti, come anche la rimozione di funzionalità obsolete. Se il tuo progetto non fnziona come dovrebbe dopo l'aggiornamento alla v2.0, consulta questa guida per una panoramica di tutti i cambiamenti e le istruzioni su come aggiornare il tuo codice.
+
+Vedi il [registro delle modifiche](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) per le note di rilascio complete.
+
+### Rimosso: Supporto di Node 14
+
+È previsto che nell'Aprile 2023 Node 14 concluderà il suo ciclo di vita.
+
+Astro v2.0 abbandona il supporto di Node 14 interamente, affinchè gli utenti di Astro possano trarre vantaggio dalle funzionalità più moderne di Node.
```

Comment:
**nitpick:** I would rephrase this as `Astro v2.0 non supporterà più Node 14, in modo che gli utenti` ...